stable/grafana/values.yaml

```diff
@@ -475,7 +475,7 @@ smtp:
 sidecar:
   image:
     repository: kiwigrid/k8s-sidecar
-    tag: 0.1.151
+    tag: 1.3.0
```

Comment:
kind of a massive step.

Reply:
Yeah - I've tried to do my best to validate no breakage by looking at code upstream... and I'm pretty sure we're all good here. Note that it's not as huge as you think: `0.1.151` was last pre-1.0 tag and then only 2 releases post-1.0 so not huge jumps in terms of versions at least.